src/ui/public/vislib/lib/dispatch.js

```diff
@@ -101,9 +101,7 @@ export default function DispatchClass(Private, config) {
       const isClickable = this.listenerCount('click') > 0;
       const addEvent = this.addEvent;
       const $el = this.handler.el;
-      if (!this.handler.highlight) {
-        this.handler.highlight = self.highlight;
-      }
+      this.handler.highlight = self.highlight;
```

Comment:
great, thanks for flagging this. you're right, this is not going to work since we're also overriding in  other places.